packages/angular/src/builders/webpack-browser/schema.json

```diff
@@ -368,6 +368,17 @@
       "description": "Override default Angular browser builder",
       "type": "string",
       "default": "@angular-devkit/build-angular:browser"
+    },
+    "customWebpackConfig": {
+      "description": "Options for additional webpack configurations",
+      "type": "object",
+      "properties": {
+        "path": {
+          "description": "Path to additional webpack configuration. Paths will be resolved to workspace root.",
+          "type": "string"
+        }
+      },
+      "additionalProperties": false
```

Comment:
I guess we should add `path` as a required property of the `customWebpackConfig`.

Reply:
Yes good idea

Reply:
Actually no. I don't want to. I'm actually going to change that `additionalProperties` to `true`, otherwise it might be conflict with third party builders